docs/guide/twig_functions_reference.md

```diff
@@ -669,3 +671,35 @@ For a Location alias set up a 301 redirect to the Location's current URL when:
     In the back end, `ez_path()` uses the Router to generate links.
 
     This makes it also easy to generate links from PHP, via the `router` service.
+
+### `ibexa_oauth2_connect_path`
+
+#### Description
+
+`ibexa_oauth2_connect_path` is a Twig function that generates a relative path for the given [OAuth2 route](user_management/oauth.md).
+
+#### Prototype and Arguments
+
+`ibexa_oauth2_connect_path ( string $identifier [, array $parameters, bool $relative ] ): string`
+
+|Argument name|Type|Description|
+|------|------|------|
+|`identifier`|string|Identifier of the OAuth connection|
+|`parameters`|`array`|Route parameters|
+|`relative`|`boolean`|Whether to generate a relative path|
+
+### `ibexa_oauth2_connect_url`
+
+#### Description
+
+`ibexa_oauth2_connect_url` is a Twig function that generates an absolute URL for the given [OAuth2 route](user_management/oauth.md).
+
+#### Prototype and Arguments
+
+`ibexa_oauth2_connect_url ( string $identifier [, array $parameters, bool $schemeRelative ] ): string`
+
+|Argument name|Type|Description|
+|------|------|------|
+|`identifier`|string|Identifier of the OAuth connection|
+|`parameters`|`array`|Route parameters|
+|`schemeRelative`|`boolean`|Whether to generate a relative URL|
```

Comment:
```suggestion
|`schemeRelative`|`boolean`|Specifies whether to generate a relative URL|
```
Re: `boolean` spalling: https://www.php.net/manual/en/language.types.boolean.php

Reply:
Nevertheless, I disagree. Boolean is an adjective that derives from a surname and as such, should be capitalized.
We are not talking about code here - in code developers can use whatever syntax requires them to use. We are describing the nature of a variable in the documentation (we might as well get rid of the code delimiters in this column, as we are not quoting code) and I'd rather stick to what the following sources dictate:
https://www.merriam-webster.com/dictionary/Boolean
https://www.ldoceonline.com/dictionary/boolean
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/boolean